metaseq/trainer.py

```diff
@@ -1212,11 +1212,12 @@ def _prepare_sample(self, sample, is_dummy=False):
         def lower_precision(t):
```

Comment:
From the [original PR](https://github.com/facebookresearch/metaseq/pull/668/files), need to remove line 257 of this file:
`"is_moe": getattr(self.cfg.model, "moe_freq", 0) > 0,`

Reply:
whoops, missed that - added!